pokemon-showdown

```diff
@@ -22,6 +24,84 @@ try {
 	child_process.execSync('npm install --production', {stdio: 'inherit'});
 }
 
+// Check if the server is configured to use Go, and ensure the required
+// environment variables and dependencies are available if that is the case
+
+let config;
+try {
+	config = require('./config/config');
+} catch (e) {}
+
+if (config && config.golang) {
+	if (!process.env.GOPATH) {
+		console.log('The GOPATH environment variable is not set! It is required in order to run the server using Go.');
+		process.exit(0);
+	}
+	if (!process.env.GOROOT) {
```

Comment:
I know it's a bad idea to set it to something other than `/usr/local/go` or whatever the equivalent on Windows is, but Go will still complain about it. Maybe I'm remembering this from using older versions though. Using `go env GOROOT` seems like the safer option to me just to be explicit without making users having to set `$GOROOT` manually